apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApi.java

```diff
@@ -57,4 +60,10 @@ public interface V3AuthenticationApi extends AuthenticationApi, Closeable {
    @Override
    Token authenticateToken(TenantOrDomainAndCredentials<TokenCredentials> credentials);
 
+   @Named("token:create")
+   @POST
+   @ResponseParser(ParseTokenFromHttpResponse.class)
+   @MapBinder(BindAccessKeyAuthToJsonPayload.class)
+   @Override
+   AuthInfo authenticateAccessKey(TenantOrDomainAndCredentials<ApiAccessKeyCredentials> credentials);
```

Comment:
Please add the corresponding unit tests here: https://github.com/apache/jclouds/blob/master/apis/openstack-keystone/src/test/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApiMockTest.java

And the live test as well here: 
https://github.com/apache/jclouds/blob/master/apis/openstack-keystone/src/test/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApiLiveTest.java

And include in a comment in the PR the output of running the live tests.